### PR TITLE
{lib}[intel/2019a] GlobalArrays v5.7 with --enable-peigs

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
hebbe-s1.int.private - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2650 v3 @ 2.30GHz, Python 2.7.14
See https://gist.github.com/8bb2948d6261989998001ad638ddb985 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/de5107d78aa652fdcef3b91fca6e7790 for a full test report.

File: easybuild/easyconfigs/g/GlobalArrays/GlobalArrays-5.7-intel-2019a-peigs.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Includes the parallel eigen value solver (used by NWChem).
+# This requires additional external symbols during linking, so it won't work for software that doesn't use it.
+
+easyblock = 'ConfigureMake'
+name = 'GlobalArrays'
+version = '5.7'
+versionsuffix = '-peigs'
+
+homepage = 'http://hpc.pnl.gov/globalarrays'
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/g/GlobalArrays/GlobalArrays-5.7-intel-2019a-peigs.eb]: https?

Comment (boegel) [reply easybuild/easyconfigs/g/GlobalArrays/GlobalArrays-5.7-intel-2019a-peigs.eb]: :+1:

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3120.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/48c0e589f888380fadccd41922d5ffda for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2091.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1abd5295b238a64e33f1efbd969ef0c1 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2688.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/9b8587a9cd01f54f5368d44f3fc28b7e for a full test report.

**Comment by boegel** [comment]
Going in, thanks @Micket!

**Pull request merged**